clusters/build-clusters/01_cluster/machine_config/debug_mcp.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  labels:
+    custom-crio: high-pid-limit
+  name: debug
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,debug]}
```

Comment:
I [need to label the nodes](https://github.com/openshift/machine-config-operator/blob/master/docs/custom-pools.md#creating-a-custom-pool) in this ms:

> oc label node <node_name> node-role.kubernetes.io/debug=

This is the only manual step that I need to do after those objects are applied, right/

Reply:
Yeah, that's what you need to do. @enxebre Can machine-api add labels to a node by default?

Reply:
Yes, machine-api can add labels by default.  Please refer to this section: https://github.com/openshift/machine-api-operator/blob/master/FAQ.md#adding-labels-to-nodes-via-machines

Remember, labels are only added on creation of new machines, not existing machines.